lib/coherence/schema.ex

```diff
@@ -202,7 +202,7 @@ defmodule Coherence.Schema do
 
         def lock!(user, locked_at \\ Ecto.DateTime.utc) do
           IO.warn "#{inspect Config.user_schema}.lock!/1 has been deprecated. Please use Coherence.ControllerHelpers.lock!/1 instead."
-          changeset = Config.user_schema.changeset(user, %{locked_at: locked_at})
+          changeset = Config.user_schema.changeset(user, %{locked_at: locked_at, current_password: user.password})
```

Comment:
The current_password should not be required here. Please update the changeset logic so it only requires current_password validation if the password is being changed.